src/askui/agent.py

```diff
@@ -401,6 +402,8 @@ def _get_default_settings_for_act(self, model_choice: str) -> ActSettings:
 
     @override
     def _get_default_tools_for_act(self, model_choice: str) -> list[Tool]:
+        self._tools.append(ScreenSwitchTool(agent_os=self.tools.os))
```

Comment:
WDYT about including it right next to the `ExceptionTool` in the constructor? Seems easier and more consistent.